core/common/src/GeoCoordinateServices.ts

```diff
@@ -9,14 +9,46 @@
   NoGCSDefined = 100,
+  /** This value indicates that the conversion was performed out the normal use of application of either Geographic Coordinate Reference Systems.
+   *  This return value can often be treated as a warning in specific cases. For example, global imagery extent spawns the whole globe and
+   *  may far extend pass the normal area of the iModel project extent and the extent of normal use of its Geographic Coordinate Reference System.
+   *  In such case this value can be considered a warning as accuracy and precision is not expected in this specific case and approximate localization
+   *  of global imagery or other low accuracy context data is not essential far from the project.
+   *  If this status is returned for high accuracy data then it indicates that either Geographic Coordinate Reference Systems were inappropriately selected
+   *  for iModel or other geolocated data. If this is the case the status should be somehow reported so action
+   *  can be performed to verify used geolocation parameters.
+   *  In either case the returned coordinates are to be considered valid though they may be inaccurate or result in some unexpected distortion of graphical
+   *  elements.
+   */
   OutOfUsefulRange = 1,
+  /** Indicates a hard error where conversions were requested out of the area out of the mathematical capacity of the conversion process for either
+   *  Geographic Coordinate Reference Systems involved. An example could be to attempt a conversion involving a Transverse Mercator more than 60 degrees
+   *  East or West of the projection central meridian.
+   *  The values returned may or may not be valid and should be discarded.
+   */
   OutOfMathematicalDomain = 2,
+  /** Indicates that datum transformation between the two Geographic Coordinate Reference Systems could not be performed.
+   *  This error is usually the result of a datum transformation path requiring use of latitude/longitude grid shift files that could not be obtained
+   *  or installed. In this case the latitude/longitude transformation is bypassed but the remainder of the conversion process is completed.
+   *  This error can be ignored for low accuracy data but should be somehow reported so actions can be performed to provide the missing files.
+   */
   NoDatumConverter = 25,
+  /** Indicates that a problem occurred during vertical datum conversion. This may the result of the incapacity to access the
+   *  Geoid separation data or vertical datum differential data (such as used by NGVD29 to NAVD88 conversion).
+   *  The horizontal coordinates returned are valid but the elevation ordinate will be returned unchanged or partially changed.
+   *  This status should be somehow reported so actions can be performed to provide the missing information.
+   */
   VerticalDatumConvertError = 26,
+  /** General inner conversion engine error. Coordinates returned are invalid and should be discarded */
   CSMapError = 4096,
   Pending = -41556,
```

Comment:
What does "pending" mean?

Reply:
@pmconne I have no idea what pending means and I have not located anywhere where  this code could be returned. My assumption is that Barry wanted to reserve a status for pending conversion requests but such a status should be returned for the whole batch of requests not for every coordinate. I beleive there are other conventions for pending batch requests that are general. Anyway, we can either eliminate the status or leave it temporarily undocumented.

Reply:
Thanks, I found no usage either. We should eliminate it.

Reply:
The pending status was set in the frontend geo-service to mark coordinates that actually needed conversion from the backend as opposed to those solved from the cache. Review wording as I am not sure of terminology.

Reply:
Ah I see. Documentation looks clear to me.